src/common/Marker/Marker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+import css from './Marker.module.css';
+
+export interface MarkerProps {
+  width: number;
+  pointY?: number;
+  strokeColor: string;
+  strokeWidth: number;
+  textColor : string,
+  fontSize:number,
+  label:string,
+  data:any[]
+}
+
+export const Marker: FC<Partial<MarkerProps>> = ({
+  pointY = 10,
+  width,
+  strokeWidth = 2,
```

Comment:
All of these default properties should be `Marker.defaultProps = {}` so props are bound correctly.